content/usage/extensions/estafette-extensions.md

```diff
@@ -328,4 +328,52 @@ Notes:
 ```
   when:
     status == 'failed'
-```
\ No newline at end of file
+```
+
+### extensions/npm-audit
+
+```yaml
+stages:
+  audit:
+    image: extensions/npm-audit:stable
+    action: audit
+    level: low
+    dev-level: none
+    workspace: estafette
+    channels:
+    - '#builds-${ESTAFETTE_LABEL_TEAM}'
+```
+
+#### audit
+
+For now the only supported action is `audit`
+
+##### Level
```

Comment:
All headings in this page are lowercase, can you update?

Reply:
Actually the only time there are subheadings for extensions is for actions or other information than the actual properties. So not sure whether these should be headings.

Reply:
yeah, I also thought that. But then I saw: `##### cloning another repository from the same owner`